cartridge/roles/metrics.lua

```diff
@@ -118,6 +123,14 @@ local function apply_config(conf)
         end
         metrics_config_present = false
     end
+    local params, err = argparse.parse()
```

Comment:
Code duplication is not ok
Please move it to special function
And does it really necessary to call it in both `init` and `apply_config`?